### ISSUE TITLE
Pressing the smiley do not enter pairing mode

### ISSUE THREAD

**Issue by ThonyJ** [nurikk/z2m-frontend#1965]
[state.json.zip](https://github.com/nurikk/zigbee2mqtt-frontend/files/14808470/state.json.zip)
### What happened?

When I hit the smiley to the right of 'permit join' Zigbee2MQTT turn into dark mode but do not go into pairing mode. 
Enter pairing mode works if I go to Settings, tick 'permit join' and then hit Submit-button.
I am using the latest version of Zigbee2MQTT addon (1.36.0-1) in Homeassistant.
I have not tested other browsers then Microsoft Edge.

### What browsers are you seeing the problem on?

Microsoft Edge

### Relevant stacktrace

```shell
-
```

### Did you download state.json.zip?
The attached stat.json is after I pressed the smiley.

**Comment by Impact123** [comment]
Yeah well the smiley is the theme button. Press the one on the left beside it. 
![image](https://github.com/nurikk/zigbee2mqtt-frontend/assets/899193/e86fd87d-c630-404a-b042-f02653e706cb)

**Comment by ThonyJ** [comment]
> Yeah well the smiley is the theme button. Press the one on the left beside it. ![image](https://private-user-images.githubusercontent.com/899193/318136473-e86fd87d-c630-404a-b042-f02653e706cb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE3NDU4OTksIm5iZiI6MTcxMTc0NTU5OSwicGF0aCI6Ii84OTkxOTMvMzE4MTM2NDczLWU4NmZkODdkLWM2MzAtNDA0YS1iMDQyLWYwMjY1M2U3MDZjYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMyOVQyMDUzMTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wNTAzN2NiMGYzMGVmYjI1YzMwNzdlYTI4ZWJjMGQ1ZTdiYzVlZDc1OGRlYTg0Nzk1OTQwZTZmNmVkODJlZmFjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.LXZ6GthR7r7cEoXwgogOOuLGauSwxcM4ZhYFhTLvRYM)

Thank you very much Impact123. It was a long time I joined any devices devices so I had forgot.
